hoomd/data.py

```diff
@@ -782,7 +786,9 @@ def take_snapshot(self,
         in the snapshot
 
         Examples::
-
+            
+            import hoomd
+            system = hoomd.init.create_lattice(unitcell=hoomd.lattice.sc(a=2.0), n=5);
             snapshot = system.take_snapshot()
```

Comment:
Please remove this duplicated `take_snapshot` (since you are already cleaning up the docs!).